### PR TITLE
Endpoints for audiences CRUD

### PR DESCRIPTION
**Pull request by davimacedo**

### CODE REVIEW

File: src/Routers/AudiencesRouter.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    return super.handleUpdate(req);
+  }
+
+  handleDelete(req) {
+    req.params.className = '_Audience';
+    return super.handleDelete(req);
+  }
+
+  mountRoutes() {
+    this.route('GET','/push_audiences', req => { return this.handleFind(req); });
```
Comment (flovilmart) [inline src/Routers/AudiencesRouter.js]: we need to protect those for masterKey only please

File: src/Routers/AudiencesRouter.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    if (body.count) {
+      options.count = true;
+    }
+    if (body.include) {
+      options.include = String(body.include);
+    }
+
+    return rest.find(req.config, req.auth,
+      '_Audience', body.where, options, req.info.clientSDK)
```
Comment (flovilmart) [inline src/Routers/AudiencesRouter.js]: can you put it all on 1 line to match style.

**Comment by flovilmart** [comment]
@davimacedo bump? I’d like to get that in for 2.5.0!

**Comment by davimacedo** [comment]
Sorry about the delay. Done.

File: src/Routers/AudiencesRouter.js (hunk 2 above)
Comment (davimacedo) [inline src/Routers/AudiencesRouter.js]: done

File: src/Routers/AudiencesRouter.js (hunk 1 above)
Comment (davimacedo) [inline src/Routers/AudiencesRouter.js]: done

**Comment by davimacedo** [comment]
I also fixed it to work on portgres

**Comment by davimacedo** [comment]
Just added few more tests

**Comment by flovilmart** [comment]
Awesome! Let’s get that in!

**Comment by davimacedo** [comment]
Rebased, resolved the conflict and also lint problems due to https://github.com/parse-community/parse-server/commit/e94991b368203a168f48bd20390ef6e2ef733375

**Comment by natanrolnik** [comment]
Good! Thanks @davimacedo. 
Tests are failing apparently.

**Comment by natanrolnik** [comment]
Did you run `npn run lint` locally?

**Comment by davimacedo** [comment]
Fixed. Yes. I have run all tests before pushing, but I had not yet updated the node modules. The new modules of the last commit are requiring new lint rules.

**Pull request merged**